### PR TITLE
Package completions for aliased imports

### PR DESCRIPTION
**Pull request by bthesorceror**

### CODE REVIEW

File: src/handlers/completion.rs [hunk 1]
```diff
@@ -241,6 +241,7 @@ async fn find_completions(
                     get_user_matches(info, ctx).await?;
 
                 items.append(&mut user_matches);
+                // Find imports aliases
```
Comment (jsteenb2) [inline src/handlers/completion.rs:244]: can you help me understand which line this comment applies too? Is it the `user_matches`?

File: src/visitors/semantic/completion/results.rs [hunk 2]
```diff
@@ -0,0 +1,240 @@
+use crate::protocol::responses::{
+    CompletionItem, CompletionItemKind, InsertTextFormat,
+};
+use crate::shared::signatures::get_argument_names;
```
Comment (jsteenb2) [inline src/visitors/semantic/completion/results.rs:4]: is there anything new here? I'm having difficulty finding diffs between the old and new 😬

File: src/handlers/completion.rs (hunk 1 above)
Comment (bthesorceror) [reply src/handlers/completion.rs:244]: 😬 sorry that was a planning comment I forgot to remove

File: src/visitors/semantic/completion/results.rs (hunk 2 above)
Comment (bthesorceror) [reply src/visitors/semantic/completion/results.rs:4]: nothing new, just breaking up some large files into their own modules

Comment (jsteenb2) [reply src/visitors/semantic/completion/results.rs:4]: sweet, thanks for the heads up 👍

**Pull request merged**